### PR TITLE
DecodedToStream to DecodeToStream

### PR DESCRIPTION
**Pull request by MikeAlhayek**

### CODE REVIEW

File: src/OrchardCore/OrchardCore.Abstractions/Base64.cs [hunk 1]
```diff
@@ -28,14 +28,18 @@ public static string FromUTF8Base64String(string base64)
         return Encoding.UTF8.GetString(span.Slice(0, bytesWritten));
     }
 
+    [Obsolete("This will be deprecated in v4. Please use DecodeToStream instead.")]
+    public static Stream DecodedToStream1(string base64)
```
Comment (sebastienros) [inline src/OrchardCore/OrchardCore.Abstractions/Base64.cs:32]:
```suggestion
    public static Stream DecodedToStream(string base64)
```

File: src/OrchardCore/OrchardCore.Infrastructure/Scripting/CommonGeneratorMethods.cs [hunk 2]
```diff
@@ -41,7 +41,8 @@ static CommonGeneratorMethods()
         Name = "gzip",
         Method = serviceProvider => (Func<string, string>)(encoded =>
         {
-            var compressedStream = Base64.DecodedToStream(encoded);
+            var compressedStream = Base64.DecodeToStream(encoded);
+            compressedStream.Seek(0, SeekOrigin.Begin);
```
Comment (sebastienros) [inline src/OrchardCore/OrchardCore.Infrastructure/Scripting/CommonGeneratorMethods.cs:45]: Why? the DecodeToStream method is now doing it

Comment (sebastienros) [reply src/OrchardCore/OrchardCore.Infrastructure/Scripting/CommonGeneratorMethods.cs:45]: Looks like a merge issue as this is what the other PR removed

Comment (MikeAlhayek) [reply src/OrchardCore/OrchardCore.Infrastructure/Scripting/CommonGeneratorMethods.cs:45]: the Other PR added this "compressedStream.Seek(0, SeekOrigin.Begin)"

Comment (MikeAlhayek) [reply src/OrchardCore/OrchardCore.Infrastructure/Scripting/CommonGeneratorMethods.cs:45]: wait! how did this show up here.

Comment (sebastienros) [reply src/OrchardCore/OrchardCore.Infrastructure/Scripting/CommonGeneratorMethods.cs]: It added it, not there

Comment (MikeAlhayek) [reply src/OrchardCore/OrchardCore.Infrastructure/Scripting/CommonGeneratorMethods.cs]: @sebastienros I removed it. You good with this?

**Review by sebastienros — APPROVED** [review]

**Pull request merged**